packages/vite-node/src/externalize.ts

```diff
@@ -11,6 +11,9 @@ const ESM_SYNTAX_RE
 const ESM_EXT_RE = /\.(es|esm|esm-browser|esm-bundler|es6|module)\.js$/
 const ESM_FOLDER_RE = /\/(es|esm)\/(.*\.js)$/
 
+// https://stackoverflow.com/a/15123777
+const COMMENT_RE = /\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm
+
```

Comment:
Would https://github.com/antfu/strip-literal be better here? It should be able to cover all kinds of edge cases.

Reply:
Yeah, that would be an option, but that's essentially running js-token based lexer, so if we go with something more sophisticated, it would be probably more direct (and also faster?) to use es-module-lexer's esm detection https://github.com/guybedford/es-module-lexer?tab=readme-ov-file#esm-detection

My impression is that over-stripping the code is mostly fine, so this regex should probably suffice for our usage.